bin/create_ops_users.sh

```diff
@@ -1,25 +1,34 @@
 #!/bin/bash
 
-ACCOUNT_ID=$(aws sts get-caller-identity --query 'Account' --output text)
-aws iam create-group --group-name admins
-aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/AdministratorAccess --group-name admins
+LPASS_STATUS="$(lpass status)"
 
-aws iam create-user --user-name ops1
-aws iam add-user-to-group --user-name ops1 --group-name admins
-OPS1PWORD=$(aws secretsmanager get-random-password | jq -r '.RandomPassword')
-aws iam create-login-profile --user-name ops1 --password "$OPS1PWORD"
+if [[ "$LPASS_STATUS" =~ ^(Logged in as) ]]
```

Comment:
Can we switch this to a guard clause instead of a big if else statement? 

https://deviq.com/design-patterns/guard-clause

Also instead of checking the text that gets returned we should validate the exit code for `lpass status` the script itself should return an error code (non-zero value) so that any script that gets called by this fails. 

I think it should be something like 

```bash
lpass status
if [ $? -ne 0]; then 
  echo "Error: Must be logged in lpass"
  exit 1
fi
```

Reply:
For sure!